### ISSUE TITLE
env command clearing ENV vars set by pkgx

### ISSUE THREAD

**Issue by sporkd** [pkgxdev/pkgx#844]
Another little bug with pkgx ENV var functionality (Possibly related to #841)

ENV vars that were set by by pkgx.yaml are getting cleared out after running the [`env`](https://github.com/pkgxdev/pkgx/blob/46205c45125a8fc1c82abe7c1862e745a1aeb7e1/src/modes/shellcode.ts#L80) command alone with no args. See session below:

<img width="548" alt="Screenshot 2023-10-29 at 3 09 36 PM" src="https://github.com/pkgxdev/pkgx/assets/11115/96d6d876-0358-480c-9ed1-ec652e137be7">

### Note:
Strangely enough, it's NOT cleared out when I run `env | grep TEMP` multiple times above. Only when I run `env` alone (or piped to /dev/null as above to hide output).

My suspicion is it's related to this [command shellcode](https://github.com/pkgxdev/pkgx/blob/46205c45125a8fc1c82abe7c1862e745a1aeb7e1/src/modes/shellcode.ts#L80) being run.

**Comment by mxcl** [comment]
Yeah I've been waiting for this report. Will fix in next patch release.

**Comment by mxcl** [comment]
We no longer do `env +foo`
